tox.ini

```diff
@@ -37,3 +37,9 @@ deps =
     pylint
 commands =
     pylint imagemodal/
+
+[testenv:transifex]
+deps =
+    transifex-client
+commands =
+    tx push -s
```

Comment:
We can flesh this more out if/when we need.

I used this and it worked for the initial upload.

As it were, I actually also enabled a Transifex/Github integration
that will _hopefully_ push/pull/PR strings automatically.

I haven't been able to test it yet, though, until after this is live.

Reply:
TODO: I should probably include a pull command..